### DISCUSSION TITLE
Can you give me some advise to build in/outflow boundary conditions?

### DISCUSSION THREAD

**Discussion by qwerty12345697** [Anura3D/Anura3D_OpenSource#179] [Q&A]
I want to achieve the same condition as the paper(Simulation of Soil Fluidization Around a Pressurized Leaking Pipe Using the Material Point Method). Can you give me some advise to build in/outflow boundary conditions?

**Comment by murphy-john** [comment]
https://www.sciencedirect.com/science/article/abs/pii/S004579301830728X?casa_token=utTlVIWUXIYAAAAA:tFbLXMjyu8XMSdL7bkfxE2lkW6ys4e5oE5wgc_DuTznAkrEzlp9VwztsErg9ezh4fEPaYRECHlQ

**Comment by FrancescaCeccato** [comment]
You can probaly check this discussion: #174
